src/read_params.jl

```diff
@@ -8,6 +8,9 @@ export  fill_defaults!
 
 """ Recursively updates nested dictionaries with default parameters"""
 function fill_defaults!(params::Dict; param_key = "fit_params")
+    # THIS IS THE LINE FROM THE polydeg-per-N BRANCH - HOW SHOULD THIS BE 
```

Comment:
@gelzinyte --- I had to resolve a merge conflict here, but didn't know how to correctly merge. Can you please take a look at this?

Reply:
everything from both branches should have been kept - I've pushed the correct list